src/services/user-admin.ts

```diff
@@ -95,3 +100,44 @@ export function addUser(sub: string): Promise<void> {
             throw reason;
         });
 }
+
+export function getAnnouncements(): Promise<AnnouncementServerData[]> {
```

Comment:
Can we catch errors and display snackBar ?

Reply:
Yes I forgot you're right